scanner/src/main/kotlin/scanners/fossid/FossIdUrlProvider.kt

```diff
@@ -43,6 +43,9 @@ internal class FossIdUrlProvider private constructor(
     private val urlMapping: Map<Regex, String>
 ) {
     companion object : Logging {
+        /** The prefix of option keys that define a URL mapping. */
+        internal const val PREFIX_URL_MAPPING = "map"
```

Comment:
Should we be more verbose here, like "urlMapping" or even "repo(sitory)UrlMapping"?

Reply:
Changed the prefix to "urlMapping".